src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const trimmedStyles = sourceString.split(';').map((style) => style.trim());
+
+  const clearedStyles = trimmedStyles
```

Comment:
What does reduce return? Can u return it Immediately?
```suggestion
  return trimmedStyles
```